ddtrace.gemspec

```diff
@@ -67,5 +67,7 @@ Gem::Specification.new do |spec|
   # Used by profiling
   spec.add_dependency 'libddprof', '~> 0.6.0.1.0'
 
+  spec.add_dependency 'ddsketch'
```

Comment:
![dancing](https://c.tenor.com/M33cQIAC6yQAAAAC/penguin-cute.gif)

Reply:
We should version `ddsketch` here, so we don't break users of, for example, `ddtrace 1.2.0` when `ddsketch 1.0` is released. 

I recommend `spec.add_dependency 'ddsketch', '~> 0.1'` at this very moment.